sdks/java/io/cassandra/build.gradle

```diff
@@ -29,6 +29,13 @@ enableJavaPerformanceTesting()
 description = "Apache Beam :: SDKs :: Java :: IO :: Cassandra"
 ext.summary = "IO to read and write with Apache Cassandra database"
 
+def cassandra_version = "3.5.0"
```

Comment:
Just minor comment because we fix this one recently, this one is unneeded (we unified Cassandra versions (of hadoop-format and the cassandra module) in the parent dependency.

Reply:
+1, thx